make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
I am curious how is `hidden-source-map` useful? There doesn't seem to be general support for it. Do you know about some external tool that can work it it? Otherwise it could be no source map emitted as well.